doc/source/_static/js/landing_page.js

```diff
@@ -67,29 +70,39 @@ function displayFamilies(familyCounts) {
     familyName.className = "family-name";
     familyName.textContent = family;
 
-    const familyCountElem = document.createElement("span");
-    familyCountElem.className = "family-count";
-    familyCountElem.textContent = `(${familyCount})`;
-
     familyRow.appendChild(checkbox);
     familyRow.appendChild(familyName);
-    //familyRow.appendChild(familyCountElem);
 
     familiesContainer.appendChild(familyRow);
   });
+
+  // Handle "Show more" click
+  const showMoreButton = document.querySelector(".product-families .show-more");
+  showMoreButton.addEventListener("click", () => {
+    if (!showMoreClicked) {
+      document
+        .querySelectorAll(".family-row")
+        .forEach((row) => (row.style.display = "flex"));
+      showMoreButton.style.display = "none"; // Hide "Show more" after clicking
+      showMoreClicked = true;
+    }
+  });
```

Comment:
i dont see you are handling the show less here. Instead of handling the showless in another function, use the text context. 
```suggestion
  const showMoreButton = document.querySelector(".product-families .show-more");
  const rows = document.querySelectorAll(".family-row");
  showMoreButton.addEventListener("click", () => {
    if (showMoreClicked) {
      rows.forEach((row, index) => {
        if (index >= maxVisible) {
          row.style.display = "none";
        }
      });
      showMoreButton.textContent = "Show more";
      showMoreClicked = false;
    } else {
      rows.forEach((row) => {
        row.style.display = "flex";
      });
      showMoreButton.textContent = "Show less";
      showMoreClicked = true;
    }
  });
}
```

Reply:
<img width="214" alt="Screenshot 2025-02-05 at 08 59 32" src="https://github.com/user-attachments/assets/e79c4620-7d1b-48cb-8fa1-90bcb7b41b3d" />
<img width="160" alt="Screenshot 2025-02-05 at 08 59 41" src="https://github.com/user-attachments/assets/36589379-de7c-49a6-8545-83502c1c6e74" />

Reply:
Thanks for the review and suggestion Revathy!
Very much appreciated.